draft-ietf-quic-spin-exp.md

```diff
@@ -251,7 +262,23 @@ same as those for passive RTT measurement in general. It has been shown
 {{PAM-RTT}} that RTT measurements do not provide more information for
 geolocation than is available in the most basic, freely-available IP address
 based location databases. The risk of exposure of per-flow network RTT to
-on-path devices is therefore negligible.
+on-path devices is in most cases negligible.
+
+There is however an exception, when parts of the path from client to server
+are hidden from observers. An example would be a server accessed through a
+proxy. The spin bit allows for measurement of the end-to-end
+RTT, and will thus enable adversaries near the client to discover that
+the connection does not terminate at the visible destination address, and
+that the client is probably accessing a hidden server through a proxy.
+
+Clients that want to hide their use of a proxy or a relay will want to
```

Comment:
```suggestion
Endpoints that want to hide their use of a proxy or a relay will want to
```

I don't think that we can assume that it is only clients that will be affected by this.